admin/spam/views.py

```diff
@@ -101,24 +98,38 @@ def post(self, request, *args, **kwargs):
         # return super(SpamDetail, self).post(request, *args, **kwargs)
 
     def get_context_data(self, **kwargs):
-        self.item = Comment.load(self.kwargs.get('spam_id', None))
+        item = Comment.load(self.kwargs.get('spam_id', None))
         kwargs = super(SpamDetail, self).get_context_data(**kwargs)
         kwargs.setdefault('page_number', self.request.GET.get('page', 1))
-        kwargs.setdefault('comment', serialize_comment(self.item))
+        kwargs.setdefault('comment', serialize_comment(item))
         kwargs.setdefault('status', self.request.GET.get('status', u'1'))
         return kwargs
 
     def form_valid(self, form):
+        spam_id = self.kwargs.get('spam_id', 'no find')
+        item = Comment.load(spam_id)
         if int(form.cleaned_data.get('confirm')) == Comment.SPAM:
-            self.item.confirm_spam(save=True)
+            item.confirm_spam(save=True)
+            log_message = 'Confirmed SPAM: {}'.format(spam_id)
+            log_action = CONFIRM_SPAM
         else:
-            self.item.confirm_ham(save=True)
+            item.confirm_ham(save=True)
+            log_message = 'Confirmed HAM: {}'.format(spam_id)
```

Comment:
These logs are only visible in the admin interface, correct? We have no plans of ever attaching them in a user-facing way?